app/code/Magento/User/Test/Integration/Console/Command/AssignAdminToRoleCommandTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\User\Test\Integration\Console\Command;
+
+use InvalidArgumentException;
+use Magento\Authorization\Model\ResourceModel\Role as RoleResource;
+use Magento\Authorization\Model\Role;
+use Magento\Authorization\Model\RoleFactory;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\User\Console\AssignAdminToRoleCommand;
+use Magento\User\Model\AssignUserToRole;
+use Magento\User\Model\ResourceModel\User as UserResource;
+use Magento\User\Model\User;
+use Magento\User\Model\UserFactory;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Helper\HelperSet;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * Class AssignAdminToRoleCommandTest tests AssignAdminToRoleCommand
+ *
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ */
+class AssignAdminToRoleCommandTest extends TestCase
+{
+    private const USER_NAME_OPTION = '--username';
+
+    private const ROLE_ID_OPTION = '--role-id';
+
+    /**
+     * @var ObjectManagerInterface
+     */
+    private $objectManager;
+
+    /**
+     * @var CommandTester
+     */
+    private $commandTester;
+
+    protected function setUp(): void
+    {
+        $this->objectManager = Bootstrap::getObjectManager();
+        /** @var AssignAdminToRoleCommand $command */
+        $command = $this->objectManager->get(AssignAdminToRoleCommand::class);
+        $helperSet = new HelperSet([new QuestionHelper()]);
+        $command->setHelperSet($helperSet);
+        $this->commandTester = new CommandTester($command);
+    }
+
+    /**
+     * @magentoDataFixture ./../../../../app/code/Magento/User/Test/Integration/_files/test_user.php
+     * @magentoDataFixture ./../../../../app/code/Magento/Authorization/Test/Integration/_files/test_role.php
```

Comment:
You should be able  to use
```suggestion
     * @magentoDataFixture Magento/User/_files/test_user.php
     * @magentoDataFixture Magento/Authorization/_files/test_role.php
```